pull-request/src/app.js

```diff
@@ -7,7 +7,11 @@ var Person = function(first, last, middle) {
 Person.prototype = {
 
     whoAreYou : function() {
-        return this.first + (this.middle ? ' ' + this.middle: '') + ' ' + this.last;
+        if (this.first === NaN) {
+          return this.last;
+        } else {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Remove this code after the "return" statement. ([javascript:UnreachableCode](http://localhost:9750/coding_rules#rule_key=javascript%3AUnreachableCode))

[See it in SonarQube](http://localhost:9750/project/issues?id=training%3Apull-request&pullRequest=25&issues=AWgPfv8Q6PH_c9BW1DXn&open=AWgPfv8Q6PH_c9BW1DXn)

[ISSUE_KEY]: AWgPfv8Q6PH_c9BW1DXn